code/backend/batch/utilities/helpers/LLMHelper.py

```diff
@@ -1,28 +1,31 @@
 from openai import AzureOpenAI
-from typing import List
-from langchain_openai import AzureChatOpenAI
-from langchain_openai import AzureOpenAIEmbeddings
+from typing import cast
+from langchain_openai import AzureChatOpenAI, AzureOpenAIEmbeddings
 from langchain.callbacks.streaming_stdout import StreamingStdOutCallbackHandler
+from semantic_kernel.connectors.ai.open_ai import AzureChatCompletion
+from semantic_kernel.connectors.ai.open_ai.prompt_execution_settings.azure_chat_prompt_execution_settings import (
+    AzureChatPromptExecutionSettings,
+)
 from .EnvHelper import EnvHelper
 
 
 class LLMHelper:
     def __init__(self):
         self.env_helper: EnvHelper = EnvHelper()
-        self.auth_type = self.env_helper.AZURE_AUTH_TYPE
+        self.auth_type_keys = self.env_helper.is_auth_type_keys()
         self.token_provider = self.env_helper.AZURE_TOKEN_PROVIDER
 
-        if self.auth_type == "rbac":
+        if self.auth_type_keys:
```

Comment:
took me a while to realise it was just negated! :-D (feel free to resolve!)

Reply:
Hahah yeah I had it as `self.auth_type_rbac = not self.env_helper.is_auth_type_keys()` first so there were fewer changes, but realised that the negation is kind of confusing and unnecessary